src/Lean/MetavarContext.lean

```diff
@@ -218,6 +218,12 @@ very simple unification and/or non-nested TC. So, if the "app builder" becomes a
 we may solve the issue by implementing `isDefEqCheap` that never invokes TC and uses tmp metavars.
 -/
 
+/--
+The `LocalInstance` stores information necessary for user facing interactions:
+it stores the user facing name of the typeclass in `className`, and the
+concrete typeclass instance in `fvar`. Note that the kernel does not care about
+this information, since typeclasses are entirely eliminated during elaboration.
```

Comment:
I'm actually not sure why this comment is here specifically. The kernel doesn't care about *anything* in this module.

Reply:
I suppose it's fine to contrast it with `LocalContext`